client/ayon_core/hosts/substancepainter/plugins/load/load_mesh.py

```diff
@@ -1,23 +1,130 @@
+from qtpy import QtWidgets, QtCore
 from ayon_core.pipeline import (
     load,
     get_representation_path,
 )
-from ayon_core.lib import EnumDef
 from ayon_core.pipeline.load import LoadError
 from ayon_core.hosts.substancepainter.api.pipeline import (
     imprint_container,
     set_container_metadata,
     remove_container_metadata
 )
-from ayon_core.hosts.substancepainter.api.lib import (
-    parse_substance_attributes_setting,
-    parse_subst_attrs_reloading_mesh
-)
-
 
 import substance_painter.project
 
 
+def _convert(subst_attr):
+    """Function to convert substance C++ objects to python instance.
+    It is made to avoid any possible ValueError when C++ objects casting
+    as python instance.
+
+    Args:
+        subst_attr (str): Substance attributes
+
+    Raises:
+        ValueError: Raise Error when unsupported Substance
+            Project was detected
+
+    Returns:
+        python instance: converted python instance of the C++ objects.
+    """
+    if subst_attr in {"Default", "UVTile", "TextureSetPerUVTile"}:
+        return getattr(substance_painter.project.ProjectWorkflow, subst_attr)
+    elif subst_attr in {"PerFragment", "PerVertex"}:
+        return getattr(substance_painter.project.TangentSpace, subst_attr)
+    elif subst_attr in {"DirectX", "OpenGL"}:
+        return getattr(substance_painter.project.NormalMapFormat, subst_attr)
+    else:
+        raise ValueError(
+            f"Unsupported Substance Objects: {subst_attr}")
```

Comment:
I think you've now still kept the old issue that it's unclear that `Default` refers to ProjectWorkflow and that `OpenGL` refers to `NormalMapFormat` when passing it to this function. I think we're better off passing it explicit strings like `ProjectWorkflow.Default` and `NormalMapFormat.OpenGL`.

Reply:
But if I have done something like you mentioned, doesn't it mean the same setting in AYON turned out to be ProjectWorkflow.Default? If yes, this is becoming really abstract for user to recognize what they want to set up.

Reply:
> But if I have done something like you mentioned, doesn't it (e.g. Default) mean the same setting in AYON turned out to be ProjectWorkflow.Default? If yes, this is becoming really abstract for user to recognize what they want to set up.

The backend value stored in the settings, etc. - yes. But the labels on the enums or settings could be `Default` just fine. But in the code we would always be working with the explicit strings - that's what I'm saying.